### DISCUSSION TITLE
OpenDTU-OnBattery Stromzähler mit Tasmota einbinden

### DISCUSSION THREAD

**Discussion by Odiug68** [hoylabs/OpenDTU-OnBattery#1412] [Q&A] [answered]
Hallo, and welcome everone...
I have severe trouble to get my power meter info via HTTP(S) *+ JSON into my OpenDTU.

I use a TASMOTA with a IR head to pin 3 and this is working fine and as is should be.
I use the following script and the indications on the TASMOTA web interface are correct.

>D
>B

=>sensor53 r
>M 1
+1,3,s,0,9600,
1,77070100010800ff@1000,Total consumption,kWh,total_in,2
1,77070100020800ff@1000,Total feed-in,kWh,total_out,2
1,77070100100700ff@1,Power,W,power_curr,0
#

Also, the answer on the URL GET request seems to be correct.

GET >>>> http://192.168.178.47/cm?cmnd=status%208
Reveive on browser  >>>> {"StatusSNS":{"Time":"2024-11-20T13:49:13","":{"total_in":3100.47,"total_out":914.81,"power_curr":2405}}}

My problem is that I do not manage to create the correct JSON path...
I tried for hours now but do nto find any working solution ...

What is incorrect here ?
StatusSNS//power_curr

![image](https://github.com/user-attachments/assets/d8ee7255-cff6-441c-81c7-1211cf910c42)

I hope someone can help...

**Comment by andreas70** [comment]
Try StatusSNS//power_curr

**Reply by Odiug68** [reply]
Thanks Andreas,
I have tried this .... but.... answer is ..

[HttpPowerMeter] Unable to access JSON key 'power_curr' (JSON path 'StatusSNS//power_curr', position 11)

**Reply by Odiug68** [reply]
![image](https://github.com/user-attachments/assets/cd7ce707-fb24-44e5-a90e-f3505a6ce841)

**Reply by spcqike** [reply, marked as answer]
You need to give your meter a name. Whatever, doesn’t matter. But your json must look like 

StatusSNS/yourName/power_curr

try

+1,3,s,0,9600,yourName

**Reply by Odiug68** [reply]
THANKS !!!!!!
This was the problem! I had tried to name the TASMOTA in the TASMOTA web editor but this never had an effect.
With your hint I now edit the script and YES !!!
Thanks again,
![image](https://github.com/user-attachments/assets/6f265231-ec06-4838-b938-08585af9c966)
